.travis.yml

```diff
@@ -22,6 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
+before_install:
+  - gem install bundler
```

Comment:
With the recent Bundler versions `tail` is OK IMHO, and that's what we have in our `Gemfile.lock`

There is no need to deal with it in production first. Let's merge this IMHO I opened an issue in `ofn-admin` to cover this in production https://github.com/openfoodfoundation/ofn-install/issues/74